shared/teams/new-team/index.js

```diff
@@ -6,6 +6,37 @@ import {globalColors, globalMargins, globalStyles} from '../../styles'
 
 import type {Props} from './'
 
+// This logic is copied from go/protocol/keybase1/extras.go.
+
+const validTeamnamePart = (s: string): boolean => {
+  if (s.length < 2 || s.length > 16) {
+    return false
+  }
+
+  return /^([a-zA-Z0-9][a-zA-Z0-9_]?)+$/.test(s)
+}
+
+const validTeamname = (s: string): boolean => {
+  return s.split('.').every(validTeamnamePart)
+}
+
+const headerText = (errorText: string, name: string): string => {
+  if (errorText) {
+    return errorText
+  }
+
+  const i = name.lastIndexOf('.')
```

Comment:
This is throwing `cannot read property 'lastIndexOf' of undefined` on opening the top-level create a team dialog. Maybe type `name: ?Constants.Teamname`?

Reply:
Ah, thanks for catching that! Fixed by changing `props => props.name` to `props => props.name || ''`